nacc/uds3/clsform.py

```diff
@@ -65,30 +66,30 @@ def add_cls(record, packet, forms):
         pct_spn = int(record['eng_percentage_spanish'])
     except ValueError:
         raise Exception(
-            "Could not parse packet as eng_percentage_spanish is not an "
+            "[WARNING] eng_percentage_spanish is not an "
```

Comment:
I noticed you changed the text of the warning message above to have this `[WARNING]` prefix. Here, however, it's part of an exception. Did you mean to do that?

Reply:
For this and the similar issue below, I am going to switch it to just printing as a warning instead of an error. These values will be blank on the CLS form when uploaded, but NACC should catch that as an error. My hesitation is that it will appear as if the form was simply not completed instead of completed incorrectly, but ultimately I think people will be more likely to check on the packet if it is uploaded with potentially misleading errors. The warning message will still print in the log, so reviewing that will give insight.